src/storage/src/streaming_table/state_table.rs

```diff
@@ -406,12 +416,67 @@ impl<S: StateStore> StateTable<S> {
         for (pk, row_op) in buffer {
             match row_op {
                 RowOp::Insert(row) => {
+                    if ENABLE_STATE_TABLE_SANITY_CHECK && !self.disable_sanity_check {
+                        // If we want to insert a row, it should not exist in storage.
+                        let storage_row = self
+                            .keyspace
+                            .get(&pk, false, self.get_read_option(epoch))
```

Comment:
Not sure can we just let `check_bloom_filter=false` here, cc @Li0k

Reply:
If we set it  `false`, nothing will be filtered. For `SANITY_CHECK`, it looks work.

For further,  we can refer to the implementation of `get_row` and use `self.dist_key_indices == key_indices` to determine whether to use `check_bloom_filter`

Reply:
I see, in the new `streaming_table`,  we will use `keyspace.get()`  for sanity check, instead of using `get_row()`, so let's just set it `false`.